config/jobs/cert-manager/cert-manager/master/cert-manager-master.yaml

```diff
@@ -1,6 +1,6 @@
 # THIS FILE HAS BEEN AUTOMATICALLY GENERATED
 # Don't manually edit it; instead edit the "cmrel" tool which generated it
-# Generated with: cmrel generate-prow --branch=master
+# Generated with: cmrel generate-prow --branch * -o file
```

Comment:
How come this changed? Probably we want to keep `--branch=master` as that makes it obvious that this was generated for master. Same in other places

Reply:
@JoshVanL altered the script to generate all branches when providing the "*" value.
see https://github.com/cert-manager/release/pull/89